Ustring.php

```diff
@@ -220,6 +212,26 @@ public function __construct($string = null)
     }
 
     /**
+     * Check if ext/mbstring is available.
+     *
+     * @return  bool
+     */
+    public static function checkMbString()
+    {
+        return function_exists('mb_substr');
```

Comment:
Ok but I can't find any function definition for `mb_substr` there, only a static method somewhere in a shim. If i did not miss anything, I think the check you perform here won't work with https://github.com/nicolas-grekas/Patchwork-UTF8